### DISCUSSION TITLE
fastfetch with custom ascii art generated by command

### DISCUSSION THREAD

**Discussion by batlopes** [fastfetch-cli/fastfetch#1397] [Q&A] [answered]
I would like to know if there is any way to use a custom logo, since this logo is a random ascii art generated by another system command.
The only way I have been able to use this random ascii art generated, so far, was by saving it to a file and passing this file to fastfetch. But I didn't think it was cool to have to save it to a file, since this will always be done, since it is a random art.

The script that is generating the ascii art is: [https://gitlab.com/phoneybadger/pokemon-colorscripts](https://gitlab.com/phoneybadger/pokemon-colorscripts)

**Comment by CarterLi** [comment, marked as answer]
https://github.com/fastfetch-cli/fastfetch/wiki/Logo-options#file-raw

**Reply by batlopes** [reply]
It worked! Thank you very much!

**Comment by Jinra** [comment]
How can this be done using the config file? I've tried a bunch of variations but can't get it working (only works through CLI args)

**Reply by CarterLi** [reply]
A new logo type `command-raw` was added later on. You may use it. https://github.com/fastfetch-cli/fastfetch/wiki/Logo-options#command-raw

**Reply by Jinra** [reply]
works perfectly, thanks!
